frontend/src/Pages/Calculator/Calculator.js

```diff
@@ -1,138 +1,180 @@
 import React from "react";
 import { useState } from "react";
+import { Formik, Field, Form } from "formik";
 
 export function Calculator() {
-  const [age, setAge] = useState(21);
-  const [weight, setWeight] = useState(150);
-  const [height, setHeight] = useState(60);
+  const [bmr, setBmr] = useState(0);
+  const [tdee, setTdee] = useState(0);
+  const [calories, setCalories] = useState(0);
+  const [protein, setProtein] = useState(0);
+  const [fats, setFats] = useState(0);
+  const [carbs, setCarbs] = useState(0);
 
-  function handleAge(e) {
-    setAge(e.target.value);
-  }
+  function calculateBMR({age, weight, height, gender, alevel, goal, units}) {
+    let calculations = 0;
```

Comment:
This would be a great place to add a comment linking to the methodology you're using here.